com.unity.ml-agents/Runtime/Inference/ModelRunner.cs

```diff
@@ -76,9 +74,10 @@ internal class ModelRunner
                 // D.logEnabled = m_Verbose;
 
                 sentisModel = ModelLoader.Load(model);
+                sentisModelInfo = new SentisModelInfo(sentisModel, deterministicInference);
```

Comment:
you could do
`using var new SentisModelInfo(`
To avoid having to call the dispose manually at the end (and which may not be called if an exception is thrown)

Reply:
ah. good point. will change.